sematic/db/schema.sql.sqlite

```diff
@@ -10,7 +10,7 @@ CREATE TABLE runs (
     ended_at timestamp,
     resolved_at timestamp,
     failed_at timestamp,
-    parent_id character(32), description TEXT, tags TEXT, source_code TEXT, root_id character(32), nested_future_id character(32), exception TEXT,
+    parent_id character(32), description TEXT, tags TEXT, source_code TEXT, root_id character(32), nested_future_id character(32), exception TEXT, external_jobs_json_encodable JSONB, resource_requirements_json JSONB,
```

Comment:
What is `external_jobs_json_encodable`?

Reply:
ah, changes from another branch got persisted in the schema on this branch. Will fix.